File: src/main/java/com/itsci/mju/maebanjumpen/dto/PersonDTO.java
package com.itsci.mju.maebanjumpen.dto;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PersonDTO {
    private Integer personId;
    private String email;
    private String firstName;
    private String lastName;
    private String idCardNumber;
    private String phoneNumber;
    private String address;
    private String pictureUrl;
    private String accountStatus;
    private LoginDTO login; // Use LoginDTO

}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/person/dto/PersonDTO.kt
package com.itsci.mju.maebanjumpen.person.dto

import com.itsci.mju.maebanjumpen.login.dto.LoginDTO

data class PersonDTO(
    var id: Long? = 0,
    var email: String? = null,
    var firstName: String? = null,
    var lastName: String? = null,
    var idCardNumber: String? = null,
    var phoneNumber: String? = null,
    var address: String? = null,
    var pictureUrl: String? = null,
    var accountStatus: String? = null,
    var login: LoginDTO? = null
)

